graphql/schema.graphql

```diff
@@ -369,6 +370,12 @@ input CopyProjectInput {
   newProjectId: String
 }
 
+input MoveProjectInput {
+  id: String!
```

Comment:
this could theoretically be either a projectId or a repoId, so can we please label it more specifically?